### ISSUE TITLE
Build failure with numpy 1.17.0

### ISSUE THREAD

**Issue by toddrme2178** [nipy/nibabel#781]
I am getting the following error when running the unittests with with numpy 1.17.0.  It doesn't happen with numpy 1.16

```
======================================================================
ERROR: nibabel.tests.test_volumeutils.test_a2f_non_numeric
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/abuild/rpmbuild/BUILD/nibabel-2.4.1/nibabel/tests/test_volumeutils.py", line 702, in test_a2f_non_numeric
    arr.astype(float)
TypeError: Cannot cast array from dtype([('f1', '<f4'), ('f2', '<i2')]) to dtype('float64') according to the rule 'unsafe'
```

**Comment by effigies** [comment]
Yup, this will be fixed in the next release: #768

**Comment by toddrme2178** [comment]
Great!
